data/sql/updates/pending_db_world/rev_1618600341183458700.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618600341183458700');
+
+INSERT INTO `spell_dbc` (`ID`, `DefenseType`, `Attributes`, `SpellLevel`) VALUES
+('20253','2','8192','0');
+
+INSERT INTO `spell_dbc` (`ID`, `DefenseType`, `SpellLevel`) VALUES
+('58620','1','0'),
+('58621','1','0');
+
+INSERT INTO `spell_dbc` (`ID`, `Attributes`) VALUES
+('64382','8192'),
+('20252','8192'),
+('20616','8192'),
+('20617','8192'),
+('25272','8192'),
+('25275','8192');
```

Comment:
1) we should use the DELETE before the INSERT because this database is supposed to be pre-filled. It's empty by default but some users already have data inside since we offer a SQL version of the DBC to use instead to override the entire DBC.

2) You should not use a "partial" version of the spell, this table needs the entire spell data. Can you use the guide from @Helias [here please](https://github.com/azerothcore/azerothcore-wotlk/pull/5298#issuecomment-820585543 )

Reply:
Thanks for the conversion

Reply:
Also, don't use '' around numbers

Reply:
@Yehonal I have a question, since the references to the powers that I am adding are not in the database, should I add all other values? otherwise will they be overridden by NULL values?

Reply:
@ErnestoGonzalezC an example of a spell_dbc PR created with the Spell Editor: https://github.com/azerothcore/azerothcore-wotlk/commit/60f865fce67bac8d1a9465fc7c8b1a76967299ac

Reply:
It seems like a lot of work but I'll try

Reply:
if you follow the Helias guide should be quite simple. Once you install the spell editor you can do the change with the GUI instead of composing the query by yourself. If you need any help you can also contact Helias on Discord directly

Reply:
@Yehonal Sorry to mention it, but I can't find the attribute column in spell.dbc or similar

Reply:
AttributesEx, AttributesExA-G

Reply:
![spell_editor](https://user-images.githubusercontent.com/80600689/115090400-9deb7100-9ec9-11eb-9f70-f0da9a6c1f91.png)
thanks for the answer @kitzunu but where I am trying to look is in the wowspell editor interface.

Reply:
Should be inside the tab "Flags"

Reply:
Thanks for the help